src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/Microsoft.NET.Sdk.Razor.Components.Wasm.targets

```diff
@@ -25,6 +25,14 @@ Copyright (c) .NET Foundation. All rights reserved.
     <!-- Trimmer defaults -->
     <PublishTrimmed Condition="'$(PublishTrimmed)' == ''">true</PublishTrimmed>
     <TrimMode Condition="'$(TrimMode)' == ''">link</TrimMode>
+    <TrimmerRemoveSymbols Condition="'$(TrimmerRemoveSymbols)' == ''">false</TrimmerRemoveSymbols>
+
+    <!-- Runtime feature defaults to trim unnecessary code -->
+    <EventSourceSupport Condition="'$(EventSourceSupport)' == ''">false</EventSourceSupport>
+    <UseSystemResourceKeys Condition="'$(UseSystemResourceKeys)' == ''">true</UseSystemResourceKeys>
+    <EnableUnsafeUTF7Encoding Condition="'$(EnableUnsafeUTF7Encoding)' == ''">false</EnableUnsafeUTF7Encoding>
```

Comment:
Nit: I know it's not related to this change, but it just caught my eye... the naming is so inconsistent. We have "Something**Support**" then "**Use**Something" and then also "**Enable**Something". Would be nice to have some system to this.

Reply:
The difference between "Something**Support**" and "**Enable**Something" is what the default behavior is. By default, "EventSource", "Debugger", and "HttpActivityPropagation" are all supported by default. UTF7 Encoding is disabled by default, and you need to enable it to use it.

Since the default is enabled, it wouldn't make sense to say `EnableDebugger`. The corresponding word would be `DisableDebugger`, but I got feedback in https://github.com/dotnet/runtime/pull/37288#discussion_r433550303 to make it a positive name, and not a negative name. So I followed the same naming with EventSource and HttpActivityPropagation.

Side Note on EnableUnsafeUTF7Encoding: ILLink.Substitutions + default feature switch values don't work very well with the way we link the runtime libraries during the build. We would need a way to ignore the substitution during the runtime build, but still embed it into the assembly. I had a conversation with @sbomer about this, and we decided that for now, setting EnableUnsafeUTF7Encoding to the default value here is probably the best option for 5.0.

`UseSystemResourceKeys` is a bit of an outlier. It isn't a "support" or an "enable". It kind of fits in an "Other" category with `InvariantGlobalization`.

Any suggestions on how to make this better?

Reply:
Maybe `EnableSystemResourceTrim` - but it's not exactly the same... 
Honestly I'm just glad we obviously thought about it 😄 

Re having susbtitutions which are not applied during library build - I would have to think about this some more, but I'm definitely willing to add something to the linker to make this possible. CoreFx is special in that we ship the linked binaries, but I do expect other libraries to run linker for the purposes of analysis - so they might run into similar problem.